### PR TITLE
add lesson notifications

### PR DESCRIPTION
**Pull request by ericcf**
* copied from Moodtech, then cleaned up

[#97577342]

### CODE REVIEW

File: lib/tasks/lesson_notifications.rake [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  desc "triggers daily email/sms notifications for newly available lessons"
+  task new_lessons: :environment do
+    puts "STARTING: lesson_notifications:new_lessons task"
+    now = Time.now
+    Membership.active.each do |membership|
+      membership.task_statuses.each do |task_status|
+        next unless task_status.is_lesson? && task_status.notify_today?
+
+        participant = task_status.participant
+        puts participant.email
```
Comment (michaelwehrley) [inline lib/tasks/lesson_notifications.rake]: These `puts` methods are for log reporting correct?

**Comment by michaelwehrley** [comment]
:+1: :eyeglasses: good to me!

File: lib/tasks/lesson_notifications.rake (hunk 1 above)
Comment (ericcf) [inline lib/tasks/lesson_notifications.rake]: Yeah, it's low tech but can help with debugging if we look at the cron logs.

Comment (michaelwehrley) [inline lib/tasks/lesson_notifications.rake]: I was just wondering if there is a sentry like reporter that makes it easier to view logs, but no biggie smalls.

**Pull request merged**